README.md

```diff
@@ -24,6 +24,7 @@ export PROJECT_ID=<google-cloud project-id>
 export CLUSTER_NAME=prombench
 export ZONE=us-east1-b
 export AUTH_FILE=<path to service-account.json>
+export GOOGLE_APPLICATION_CREDENTIALS=<path to service-account.json>
```

Comment:
lets also add a comment why is this needed and that the long plan is to remove it.

Reply:
okay